certify-postgres-landregistry.properties

```diff
@@ -2,21 +2,32 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
+
 ## ------------------------------------------- Plugin Enable Properties
 mosip.certify.integration.scan-base-package=io.mosip.certify.postgresdataprovider.integration
 mosip.certify.integration.audit-plugin=LoggerAuditService
 mosip.certify.integration.data-provider-plugin=PostgresDataProviderPlugin
 mosip.certify.plugin-mode=DataProvider
 
+
+
+## --------------------------------------------DataProvider specific Certify configuration --------------------------------------------------- 
+# set custom VC expiry, supports limited subset of ISO-8601 Duration format starting from days to seconds
+# E.g. P21dT1h-2m3s for more information check javadoc of Duration.parse(String)
+mosip.certify.data-provider-plugin.vc-expiry-duration:P730d
+# Cache-Control max-age value for rendering templates
+mosip.certify.rendering-template.cache-max-age-days=1
 ## ------------------------------------------- Plugin specific usecase properties ------------------------------------------------------------
 ## Postgres DB specific config
 
 # Other supported value: [RsaSignature2018]
 mosip.certify.data-provider-plugin.issuer.vc-sign-algo=Ed25519Signature2020
 mosip.certify.vcformat.vc.expiry=true
-mosip.certify.data-provider-plugin.issuer-public-key-uri=did:web:vharsh.github.io:DID:devcp2#key-0
-mosip.certify.data-provider-plugin.issuer-uri=did:web:vharsh.github.io:DID:devcp2
+mosip.certify.data-provider-plugin.issuer-public-key-uri=did:web:inji-config.github.io:dev1:postgreslandregistry#key-0
+mosip.certify.data-provider-plugin.issuer-uri=did:web:inji-config.github.io:dev1:postgreslandregistry
```

Comment:
can we keep this to the namespace ? like did:web:inji-config.github.io:dev1:cp1 etc.. ?
Also can you send the PR to make the change in gh pages branch also ?

Reply:
??

Reply:
remove postgres keep only landregistry